tests/rbd_mirror/test_mirror_move_primary_trash_restore.py

```diff
@@ -0,0 +1,102 @@
+from tests.rbd.rbd_utils import Rbd
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """Verification of primary image having snap and clone move to trash and
+    restore back and check mirror relationship is intact.
+
+    Args:
+        **kw:
+
+    Returns:
+        0 - if test case pass
+        1 - it test case fails
+
+    Test case covered -
+    CEPH-11417 - Pool Mirror - Try deleting the primary on the relationship
```

Comment:
Make the change in polarion name reflect here as well.

Reply:
Done